openslides/agenda/config_variables.py

```diff
@@ -0,0 +1,83 @@
+from datetime import datetime
+
+from django.core.exceptions import ValidationError as DjangoValidationError
+from django.core.validators import MaxLengthValidator, MinValueValidator
+from django.utils.translation import ugettext as _
+from django.utils.translation import ugettext_lazy
+
+from openslides.core.config import ConfigVariable
+
+
+def validate_start_time(value):
+    try:
+        datetime.strptime(value, '%d.%m.%Y %H:%M')
+    except ValueError:
+        raise DjangoValidationError(_('Invalid input.'))
+
+
+def get_config_variables():
+    yield ConfigVariable(
```

Comment:
Docstring: `Generator which yields all config variables of this app. It has to be evaluated during app loading (see apps.py).`